### ISSUE TITLE
Doubt: p5.prototype.background function in settings.js

### ISSUE THREAD

**Issue by Ajayneethikannan** [processing/p5.js#3639]
#### Nature of issue?

- [x] Found a bug

#### Most appropriate sub-area of p5.js?

- [x] Core/Environment/Rendering
- [x] Image

#### Which platform were you using when you encountered this?

- [x] Mobile/Tablet (touch devices)
- [x] Desktop/Laptop
- [x] Others (specify if possible) Device independent

#### Details about the bug: 

- p5.js version:  0.7.3
- Web browser and version:  Browser independent
- Operating System:  OS independent
- Steps to reproduce this:
This code can be used for reference : 
```
let img;
function preload() {

  img = loadImage('https://i.imgur.com/VyAlcbq.jpg');
}
function setup()
{
    createCanvas(100, 100);
}
function draw() {
  rotate(5);
  background(img); //background translated leaving white space on the canvas.

  //image(img, 0, 0, this.width, this.height);
}

```

If an image is used to create background while transformations are applied before it ,
it can result in the background being translated too, instead of the background filling the screen.

This happens because in [p5.prototype.background](https://github.com/processing/p5.js/blob/master/src/color/setting.js#L176), the image function is directly called without resetting the matrix, which is handled in [p5.Renderer2D.prototype.background](https://github.com/processing/p5.js/blob/master/src/core/p5.Renderer2D.js#L46).

Also , there is a kind of duplicacy of the code , if only one of the methods are ever called. 

Should the implementation be changed that  only p5.Renderer2D.background is called , in case of image being passed as a parameter ? 
Thanks !

**Comment by lmccart** [comment]
@Ajayneethikannan I think the reason we were doing it this was way because image() is not yet implemented in webgl mode, so if you call background() with an image in that mode, it would create an error.

It looks like the image() in webgl addition is closed to being merged in #3634, eliminating this problem mentioned above. I would suggest you go ahead and make the change as you suggest, removing the redundancy, and we will merge this PR after #3634 is merged.

**Comment by Ajayneethikannan** [comment]
Sounds great ! Thanks @lmccart !
